sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -234,43 +237,43 @@ int main() {
 
 ```
 
-# Build and Test a simple SYCL program
-
-To build simple-sycl-app put `bin` and `lib` to PATHs and run following command:
-### Linux:
-   ```bash
-   export PATH=$SYCL_HOME/build/bin:$PATH
-   export LD_LIBRARY_PATH=$SYCL_HOME/build/lib:$LD_LIBRARY_PATH
-   ```
-### Windows:
-   ```bash
-   set PATH=%SYCL_HOME%\build\bin;%PATH%
-   set LIB=%SYCL_HOME%\build\lib;%LIB%
-   ```
-
-### Linux & Windows:
-   ```bash
-   clang++ -fsycl simple-sycl-app.cpp -o simple-sycl-app.exe -lOpenCL
-   ```
+To build simple-sycl-app put `bin` and `lib` to PATHs:
 
-This `simple-sycl-app.exe` application doesn't specify SYCL device for execution,
-so SYCL runtime will first try to execute on OpenCL GPU device first, if OpenCL
-GPU device is not found, it will try to run OpenCL CPU device; and if OpenCL
-CPU device is also not available, SYCL runtime will run on SYCL host device.
+**Linux**
+```bash
+export PATH=$SYCL_HOME/build/bin:$PATH
+export LD_LIBRARY_PATH=$SYCL_HOME/build/lib:$LD_LIBRARY_PATH
+```
+
+**Windows**
+```bat
+set PATH=%SYCL_HOME%\build\bin;%PATH%
+set LIB=%SYCL_HOME%\build\lib;%LIB%
+```
+
+and run following command.
+
+```bash
+clang++ -fsycl simple-sycl-app.cpp -o simple-sycl-app.exe -lOpenCL
```

Comment:
-lOpenCL is not needed in compile/link command line as soon as there are no direct OpenCL call in application.

Reply:
`-lOpenCL` is needed.

Reply:
It looks like @vladimirlaz is right.
SYCL headers do not use OpenCL API directly anymore, so there is no need to link with OpenCL library anymore!

Reply:
Do not remove -lOpenCL from here yet. I was going to do the same about a week ago, but found a case with direct call of OpenCL. I suggest run LIT + CTS without -lOpenCL first.

Reply:
As @vladimirlaz explained, we have a few test, using OpenCL API directly. We need to link OpenCL library explicitly only in this cases.
@v-klochkov, I'll update the lit tests according to updated instructions in a separate patch.